modules/rosa-create-objects.adoc

```diff
@@ -192,6 +192,9 @@ $ rosa create cluster --cluster-name=<cluster_name> [arguments]
 |--additional-control-plane-security-group-ids <sec_group_id>
 |The identifier of one or more additional security groups to use along with the default security groups that are used with the control plane nodes created alongside the cluster. For more information on additional security groups, see the requirements for _Security groups_ under _Additional resources_.
 
+|--additional-allowed-principals <arn>
+|A comma-separated list of additional allowed principal ARNs to be added to the Hosted Control Plane's VPC Endpoint Service to enable additional VPC Endpoint connection requests to be automatically accepted.
```

Comment:
```suggestion
|A comma-separated list of additional allowed principal ARNs to be added to the hosted control plane's VPC Endpoint Service to enable additional VPC Endpoint connection requests to be automatically accepted.
```
Check to make sure that "endpoint service" and "endpoint" need to be capitalized.